src/js/components/aboutTheData/AgencyDownloadLinkCell.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import kGlobalConstants from 'GlobalConstants';
+
+const AgencyDownloadLinkCell = () => (
+    <div className="agency-website">
+        <div className="usa-button-link">
+            <span className="usa-button-link__download-icon">
+                <FontAwesomeIcon
+                    data-href={`${kGlobalConstants.FILES_SERVER_BASE_URL}/agency_submissions/`}
+                    icon="file-alt" />
+            </span>
+            <a
+                target="_blank"
+                href={`${kGlobalConstants.FILES_SERVER_BASE_URL}/agency_submissions/`}
```

Comment:
Will this url be unique for each agency? It seems like we would generate it based on the agency code (which could be passed as a prop)